lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.Model.rb

```diff
@@ -633,6 +633,50 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
                                         sys_group['zones'],
                                         zone_fan_scheds)
 
+          if baseline_179d && ['Gas_Furnace', 'Electric_Furnace'].include?(system_type[0])
+            OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', "179D - For Unit Heater, adding a ZoneVentilationDesignFlowRate for outside air requirements")
+            sys_group['zones'].sort.each do |zone|
+              total_oa_m3_per_s = thermal_zone_outdoor_airflow_rate(zone)
+
+              total_oa_m3_per_m2s = total_oa_m3_per_s / zone.floorArea
+
+              next unless total_oa_m3_per_s > 0
+
+              # ventilation = model_add_zone_ventilation(model, sys_group['zones'], ventilation_type: 'Natural', flow_rate: total_oa_m3_per_s).first
+
+              tot_oa_cfm = OpenStudio.convert(total_oa_m3_per_s, 'm^3/s', 'cfm').get.round(2)
+              total_oa_cfm_per_sqft = OpenStudio.convert(total_oa_m3_per_m2s, 'm^3/m^2*s', 'cfm/ft^2').get.round(4)
+
+              OpenStudio.logFree(
+                OpenStudio::Info, 'openstudio.179D.Model',
+                "Adding zone ventilation fan for #{zone.name} - #{tot_oa_cfm} CFM total - #{total_oa_cfm_per_sqft} CFM/ft^2"
+              )
+
+              ventilation = OpenStudio::Model::ZoneVentilationDesignFlowRate.new(model)
+              ventilation.setName("#{zone.name} Ventilation")
+              ventilation.setSchedule(model.alwaysOnDiscreteSchedule)
+
+              # ventilation.setDesignFlowRate(total_oa_m3_per_s)
+              ventilation.setFlowRateperZoneFloorArea(total_oa_m3_per_m2s)
+
+              ventilation.setFanPressureRise(0.0)
+              ventilation.setFanTotalEfficiency(1.0)
+              # TODO: Should we just make this 1.0 constant, and 0 velocity?
+              ventilation.setConstantTermCoefficient(0.0)
+              ventilation.setVelocityTermCoefficient(0.224)
+              ventilation.setTemperatureTermCoefficient(0.0)
+              ventilation.setMinimumIndoorTemperature(-73.3333352760033)
+              # Should we make max = 100.0
+              ventilation.setMaximumIndoorTemperature(29.4444452244559)
+              ventilation.setDeltaTemperature(-100.0)
```

Comment:
TODO. @JanghyunJK @brianlball thoughts please. Do you agree with me, that we just make it constant and run all the time?

These values right now are taken from https://github.com/NREL/openstudio-standards/blob/cd889f3217e8ba038aa41cbb6c555ace5d38f66e/lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb#L5621-L5630 but they are meant to do the approach that the DOE prototype is taking:

* One ZV with OA requirements up to 85F indoor temp, Natural (=no fan power)
* Another ZV for 85F and up that brings in like 3.85 ACH for "non mechanical cooling"

Reply:
https://bigladdersoftware.com/epx/docs/24-1/input-output-reference/group-airflow.html#zoneventilationdesignflowrate

![image](https://github.com/user-attachments/assets/76b7daa6-111f-4385-92c4-4f950dbdc356)

A = Constant Term, B is the Temperature Term coefficient

Reply:
So currently the what I copied does:

Ventilation = Vdesign * (0.224 * |Tzone - Todb|)

And I suggest making it

Ventilation = Vdesign

Reply:
![image](https://github.com/user-attachments/assets/e89b5b16-b439-4bc1-861c-a84372e6e634)

Reply:
![image](https://github.com/user-attachments/assets/ac0df1ce-e007-4dda-83af-e57fb1ce4d00)